.vscode/launch.json

```diff
@@ -8,7 +8,7 @@
       "type": "lldb",
       "request": "launch",
       "name": "Launch",
-      "program": "${workspaceFolder}/target/debug/boa_cli",
+      "program": "${workspaceFolder}/target/debug/boa_cli.exe",
```

Comment:
I suppose unix-like systems don't need the `.exe` extension? VSCode has support for platform specific properties in launch.json: https://code.visualstudio.com/Docs/editor/debugging#_platformspecific-properties

Reply:
Yes you’re right well spotted!
I’ll change that, thanks

Reply:
Fixed!